builder/builder/Nuspec.cs

```diff
@@ -95,14 +95,14 @@ public Dependency(string id, string version)
             {
                 var p = Process.Start(
                   new ProcessStartInfo(
-                      @"..\..\..\packages\NuGet.CommandLine.5.10.0\tools\nuget.exe",
+                      @"..\..\..\packages\NuGet.CommandLine.6.1.0\tools\nuget.exe",
                       "push " + nupkgFile + " -Source https://api.nuget.org/v3/index.json -ApiKey _")
                   {
                       UseShellExecute = false,
                   });
                 p.WaitForExit();
                 if (p.ExitCode != 0) {
-                    Environment.Exit(p.ExitCode);
+                    //Environment.Exit(p.ExitCode);
```

Comment:
Just commented out to check CI runs could create all packages as github action and appveyor fail on first upload due to missing api key